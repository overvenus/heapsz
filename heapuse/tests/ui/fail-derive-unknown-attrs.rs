use heapuse_derive::Heap;

#[path = "../../examples/allowlisting.rs"]
mod allowlisting;

#[derive(Heap)]
pub struct UnknownAttr {
    #[heap_size(unknown)]
    pub a: allowlisting::FullAllow,
}

fn main() {}
