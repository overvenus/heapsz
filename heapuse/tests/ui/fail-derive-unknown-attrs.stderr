error: unknown heap_size attribute `unknown`
 --> tests/ui/fail-derive-unknown-attrs.rs:8:17
  |
8 |     #[heap_size(unknown)]
  |                 ^^^^^^^
